persistent/Database/Persist/Sql/Types.hs

```diff
@@ -25,7 +27,7 @@ data Column = Column
     , cDefault   :: !(Maybe Text)
     , cDefaultConstraintName   :: !(Maybe DBName)
     , cMaxLen    :: !(Maybe Integer)
-    , cReference :: !(Maybe (DBName, DBName)) -- table name, constraint name
+    , cReference :: !(Maybe ((DBName, DBName), FieldCascade)) -- table name, constraint name, cascade options
```

Comment:
I kinda wish this were a different datatype rather than a mash of tuples. Any opposition to writing a new datatype with field accessors instead of a tuple?